controllers/leds.py

```diff
@@ -0,0 +1,50 @@
+from components.leds import StatusLights, LedColors, DisplayType
+from components.gripper import Gripper
+from components.intake import Intake
+
+
+class LedController:
+    status_lights: StatusLights
+    gripper: Gripper
+    intake: Intake
+
+    def __init__(self) -> None:
+        ...
+
+    # is this needed?
+    def gripped_piece_is_cone(self) -> bool:
+        return False
+
+    def should_place_left(self) -> bool:
+        return False
+
+    def wants_cone(self) -> bool:
+        return False
+
+    def wants_cube(self) -> bool:
+        return False
+
+    def execute(self) -> None:
+        CONE = self.gripper.get_full_closed() and self.gripped_piece_is_cone()
+        CUBE = self.intake.is_game_piece_present()
+        LEFT = self.should_place_left()
+        RIGHT = not self.should_place_left()
+        WANTS_CONE = self.wants_cone()
+        WANTS_CUBE = self.wants_cube()
```

Comment:
I don't think these are constants. See
- https://wiki.thedropbears.org.au/sections/software/coding-standards
- https://peps.python.org/pep-0008/#function-and-variable-names
- https://peps.python.org/pep-0008/#constants